lms/static/js/inline_analytics.js

```diff
@@ -378,6 +371,23 @@ window.InlineAnalytics = (function() {
     }
 
 
+    function convertNameToArray(choiceNameString) {
+        // Convert the choice name string to a list
+        choiceNameString = choiceNameString.substring(1, choiceNameString.length -1).replace(/'/g, "");
+        return choiceNameString.split(", ");
+    }
+
+    
```

Comment:
According to http://kangax.github.io/compat-table/es5/, it works for everything, even obsolete browsers, except IE7 and 8. I think that's good enough.